core/app/sorters/spree/products/sort.rb

```diff
@@ -0,0 +1,43 @@
+module Spree
+  module Products
+    class Sort
+      def initialize(scope, params)
+        @scope = scope
+
+        @desc_order = String(params[:sort])[0] == '-'
+        @sort_field = @desc_order ? params[:sort][1..-1] : params[:sort]
+      end
+
+      def call
+        products = sort_by_updated_at(scope)
```

Comment:
I don't think there's a need to prefix method names with `sort_by` since the name of this class is `Sort` and it's pretty self-explanatory